### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by pinkmoon25**
For this project, I have Re-created the BookList app with Javascript ES6 syntax.

- Export separate modules for different functionality and Import them as required in the index.js file.
- Refactored the methods and functions to use arrow functions.
- Used Luxon library to display date and time on the page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# bookList-website
+
+This is a project to help you find and remember the best books you read.
+You can add favourite books, or books you'd like to read, you can create a book collection and this information will be stored and saved for you even if the browser is closed, you can access your collection anytime anywhere, a must have app with easy UI/UX.
+
+# Project Name : Awesome Books ♻️
+
+![screenshot}](./assets/Booklist2.png)
+
```
Comment (KDlamini) [inline README.md:9]: [OPTIONAL] The link to your screenshot is not working properly. Kindly look into it to make sure that the image is displayed as expected.

**Review by KDlamini — APPROVED** [review]

**Review by KDlamini — APPROVED** [review]
### Hi @PinkMoon25 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the project requirements for refactoring with ES6. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 
✔️  The application is working as expected
✔️  ES6 syntax is used
✔️  Functionalities are divided into reasonable modules
✔️  Good PR Description
✔️  Commit messages are meaningful
✔️  No linter errors

# Status: Approved ✔️ 

giphyapproved

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Kindly look through the comments below 👇 for optional changes**
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (KDlamini) [inline README.md]: [OPTIONAL] The link to your screenshot is not working properly. Kindly look into it to make sure that the image is displayed as expected.

Comment (pinkmoon25) [reply README.md]: Thank you @KDlamini for the review, I have updated my README to display the screenshot and the live demo

**Pull request merged**